lib/hydra/works.rb

```diff
@@ -64,21 +64,38 @@ module Works
     autoload :GenerateThumbnail,                   'hydra/works/services/generic_file/generate/thumbnail'
     autoload :PersistDerivative,                   'hydra/works/services/generic_file/persist_derivative'
 
-    # model validations
-    def self.collection? collection
+    # model validations 
+    def self.works_collection? collection
```

Comment:
These methods should be removed -- these same methods are implemented in models/concerns/work_behavior.rb, e.g.: [#works_collection](https://github.com/blancoj/hydra-works/blob/issue_164/lib/hydra/works/models/concerns/generic_work_behavior.rb#L34)